modules/utils.js

```diff
@@ -0,0 +1,34 @@
+// Hard-coded data
+const hardcodedData = [
+  'name 1',
+  'name 2',
+  'name 3',
+  'name 4',
+
+];
+
+// Fetch data function
+export async function fetchData() {
+  try {
+    // Simulate asynchronous data fetching
+    await new Promise((resolve) => setTimeout(resolve, 2000));
+
+    // Return the hardcoded data
+    return hardcodedData;
+  } catch (error) {
+    return [];
+  }
+}
```

Comment:
- [ ] Kindly use the leaderboard API to create a new game of the name of your choice as requested in the project requirement. Read the [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) to learn how to use this API.
![Screenshot from 2023-05-25 12-11-40](https://github.com/kayjeee/leaderboard-v2/assets/91142432/a1b93f57-eecb-4b19-b3d4-7d11775fcd96)